docs/internal/github_actions.rst

```diff
@@ -139,7 +140,7 @@ Test runs which only contain CMake tests are prefixed by ``cmake_``.
 
 .lst files contain a list of tests with configuration options that will turn tests on or off.
 The *test_* jobs pass in :ghfile:`tests/dcps_tests.lst`.
-Static and Release builds instead use :ghfile:`tests/static_ci_tests.lst`.
+MacOS, Windows 22, Static, and Release builds instead use :ghfile:`tests/static_ci_tests.lst`.
```

Comment:
Since `static_ci_tests.lst` is not only used for static builds, should it be renamed?

Reply:
`core_ci_tests.lst` sound okay?

Reply:
Yes